std/json.d

```diff
@@ -456,15 +490,18 @@ struct JSONValue
     ///
     /// Throws: $(D JSONException) if $(D type) is not $(D JSON_TYPE.OBJECT)
     /// or $(D JSON_TYPE.NULL).
-    void opIndexAssign(T)(auto ref T value, string key) pure
+    void opIndexAssign(T)(auto ref T value, string key) pure @safe
```

Comment:
I forgot about explicit conversions with alias this. That's a reasonable use case (while these conversions will *probably* be easy to make @safe, it's not friendly to force people to do that work). I removed the @safe from the opIndexAssign method group.